net/dnsdist/Makefile

```diff
@@ -61,12 +65,35 @@ define Package/dnsdist/install/Default
 	$(INSTALL_DIR) $(1)/etc
 	$(INSTALL_DIR) $(1)/etc/dnsdist.conf.d
 	$(INSTALL_CONF) ./files/dnsdist.conf $(1)/etc/dnsdist.conf
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/dnsdist.config $(1)/etc/config/dnsdist
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/dnsdist.init $(1)/etc/init.d/dnsdist
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dnsdist $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/usr/share/acl.d
+	$(INSTALL_DATA) ./files/dnsdist_acl.json $(1)/usr/share/acl.d
+	$(INSTALL_DIR) $(1)/usr/share/lua/dnsdist
+	$(INSTALL_DIR) $(1)/usr/share/dnsdist
+	$(CP) ./files/common.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/configuration.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/local-domains.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/os.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/dnsdist-odhcpd.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/start.lua $(1)/usr/share/lua/dnsdist/
+	$(CP) ./files/sample.uci.conf $(1)/usr/share/dnsdist/sample.uci.conf
+	$(CP) ./files/dnsdist.config $(1)/usr/share/dnsdist/simple.uci.conf
+	$(INSTALL_BIN) ./files/diag.sh $(1)/usr/share/dnsdist/diag.sh
+endef
+
+define Package/dnsdist/postinst/Default
+#!/bin/sh
+# if we are on a "real" system, IPKG_INSTROOT will be empty,
+# otherwise we are in the process of building an image and
+# thus on the host system
+local root="$${IPKG_INSTROOT}"
+
+if [ ! -e "$${root}/etc/config/dnsdist" ]; then
+        cp "$${root}/usr/share/dnsdist/simple.uci.conf" "$${root}/etc/config/dnsdist"
+fi
 endef
```

Comment:
I don't think this needs to be a postinst.  Conffile handling will ensure that a user's config is not overwritten.  The `$(INSTALL_CONF)` that was removed should work just fine.

Reply:
Thanks! Given that neither Habbie nor I can remember why we introduced this change, I'm going to sleep on it and very likely revert it tomorrow :)